decred/decred/crypto/secp256k1/curve.py

```diff
@@ -339,10 +380,18 @@ def publicKey(self, k):
 
     def parsePubKey(self, pubKeyStr):
```

Comment:
I don't think the arg is named correctly. Str is string correct? But it looks like this accepts something bytes-like.
If I'm correct, just pubKey is better.